samples/gen-go-deprecated/models/outputs.go

```diff
@@ -1,24 +1,30 @@
 package models
 
-// DefaultInternalError represents a generic 500 response.
-type DefaultInternalError struct {
-	Msg string `json:"msg"`
-}
+// BadRequest defines a response type.
+// Bad Request
+type BadRequest Error
```

Comment:
changed the name from `DefaultBadRequest` to `BadRequest`. This isn't backwards compatible, but I think I like it better, and since we're already requiring users to change their yml, this seems reasonable.